<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellDialog</class>
 <widget class="QDialog" name="SellDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new item</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: grey;
                QPushButton{	background-color: rgb(67,70,75);
                border-width: 2px;
                border-radius: 10px;
                border-color: beige;
                font: bold 12px;
                color: white;
                padding: 5px;
                }
                QPushButton:pressed {
                border-style: inset;
                border: 2px solid #add8e6;
                }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="1" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="id">
       <property name="styleSheet">
        <string notr="true">font: bold 13px;
                                    color: white;</string>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="idLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: white;
                                    border-radius: 5px;
                                    border: 2px solid light grey;
                                </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="manufacturer">
       <property name="styleSheet">
        <string notr="true">font: bold 13px;
                                    color: white;</string>
       </property>
       <property name="text">
        <string>Manufacturer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="manufacturerLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: white;
                                    border-radius: 5px;
                                    border: 2px solid light grey;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="description">
       <property name="styleSheet">
        <string notr="true">font: bold 13px;
                                    color: white;</string>
       </property>
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriptionLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: white;
                                    border-radius: 5px;
                                    border: 2px solid light grey;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="3">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="price">
       <property name="styleSheet">
        <string notr="true">font: bold 13px;
                                    color: white;</string>
       </property>
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="priceLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: white;
                                    border-radius: 5px;
                                    border: 2px solid light grey;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{	background-color: rgb(67,70,75);
                                    border-width: 2px;
                                    border-radius: 10px;
                                    border-color: beige;
                                    font: bold 12px;
                                    color: white;
                                    padding: 5px;
                                    }
                                    QPushButton:pressed {
                                    border-style: inset;
                                    border: 2px solid #add8e6;
                                    }</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{	background-color: rgb(67,70,75);
                                    border-width: 2px;
                                    border-radius: 10px;
                                    border-color: beige;
                                    font: bold 12px;
                                    color: white;
                                    padding: 5px;
                                    }
                                    QPushButton:pressed {
                                    border-style: inset;
                                    border: 2px solid #add8e6;
                                    }</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{	background-color: rgb(67,70,75);
                                    border-width: 2px;
                                    border-radius: 10px;
                                    border-color: beige;
                                    font: bold 12px;
                                    color: white;
                                    padding: 5px;
                                    }
                                    QPushButton:pressed {
                                    border-style: inset;
                                    border: 2px solid #add8e6;
                                    }</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2" rowspan="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="name">
       <property name="styleSheet">
        <string notr="true">font: bold 13px;
                                    color: white;</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: white;
                                    border-radius: 5px;
                                    border: 2px solid light grey;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="specs">
       <property name="styleSheet">
        <string notr="true">font: bold 13px;
                                    color: white;</string>
       </property>
       <property name="text">
        <string>Specs:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="specsLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: white;
                                    border-radius: 5px;
                                    border: 2px solid light grey;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: bold 18px;
                                    color: white;</string>
       </property>
       <property name="text">
        <string>Specify new phone data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
